spec/features/file_search_spec.rb

```diff
@@ -0,0 +1,55 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe 'file viewer search bar', js: true do
+  include PurlFixtures
+
+  context 'when text is entered with a file list' do
+    it 'limits shown files' do
+      stub_purl_response_with_fixture(file_purl)
+      visit_iframe_response('abc123', min_files_to_search: 1)
+      expect(page).to have_css('.sul-embed-count', count: 1)
+      expect(page).to have_css '.sul-embed-item-count', text: '1 item'
+      fill_in 'sul-embed-search-input', with: 'test'
```

Comment:
```suggestion
      fill_in 'Search this list', with: 'test'
```
Using labels helps test accessability.